themes/digital.gov/src/scss/new/guides/_guide-header.scss

```diff
@@ -0,0 +1,30 @@
+@use "uswds-core" as *;
+
+@include at-media("tablet-lg") {
+  .dg-guide__header {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+
+    &-title {
+      flex: 1;
+    }
+
+    &-image {
+      flex: 1;
+      margin-left: units(15);
+      max-width: units("mobile");
+    }
+  }
+}
+
+@include at-media("mobile") {
+  .dg-guide__header-title {
+    margin-top: units(5);
+  }
+
+  .dg-guide__header-image {
```

Comment:
Same here? If so, we should consider grouping them to avoid breaking changes in the future.

**Example**

```scss
    &-image {
      flex: 1;
      margin-left: units(15);
      max-width: units("mobile");
      
      @include at-media("mobile") {
         margin-top: units(5);
      }
    }
```